docker/images/niam/Dockerfile

```diff
@@ -44,6 +44,7 @@ ENV MONGO_HOST="192.168.99.100"
 ENV MONGO_PORT="27017"
 ENV MONGO_DB="niam"
 ENV MONGO_URI="mongodb://$MONGO_HOST:$MONGO_PORT/$MONGO_DB"
+ENV SPARQURL="http://niam.eha.io:3030/dataset/query"
```

Comment:
👍 
This approach works great as opposed to hardcoding it in the actual source code.